app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -84,7 +84,7 @@
   color: $white;
 
   &:focus {
-    color: $link-colour;
+    color: darken($link-colour, 2.5%);
```

Comment:
I think we can use `$govuk-focus-text-colour` from GOV.UK Frontend here?

Reply:
That would be inconsistent with the rest of GOV.UK. I'm not sure if this could be an intentional tradeoff for moving towards using more styles from the Design System?

Reply:
I'd be fine with a slight inconsistency within GOV.UK, since the Design System style is better and will replace the rest anyway. We're already using it here:

https://github.com/alphagov/govuk_publishing_components/blob/4a9a3267ae6b5a70af7a127bfae633bcec0ebf23/app/assets/stylesheets/govuk_publishing_components/components/_table.scss#L76-L78

Reply:
Found that we're even using the Design System style in content body (like https://www.gov.uk/vehicle-log-book), since [we use this GOV.UK Frontend mixin for govspeak](https://github.com/alphagov/govuk_publishing_components/blob/4a9a3267ae6b5a70af7a127bfae633bcec0ebf23/app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss#L80).

Reply:
Oh, interesting. I will update this then.

Reply:
Done.

Reply:
Thanks for catching this @tijmenb – indeed it's better to use this variable especially because colours (including highlight may change soon, so by doing this we're sure we're using the latest colours which have been tested for contrast), and thanks @selfthinker for this update – there are still a couple of links on gov.uk that don't rely on `govuk-frontend` hence don't have enough contrast.